app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -252,6 +227,77 @@ public Result publishSurvey(String surveyGuid, String createdOnString) throws Ex
+        return getAllSurveysMostRecentVersionV3();
+    }
+    @Deprecated
+    public Result getAllSurveysMostRecentVersion2V1() throws Exception {
+        return getAllSurveysMostRecentVersion2V3();
+    }
+    @Deprecated
+    public Result getAllSurveysMostRecentlyPublishedVersionV1() throws Exception {
+        return getAllSurveysMostRecentlyPublishedVersionV3();
+    }
+    @Deprecated
+    public Result getSurveyV1(final String surveyGuid, final String createdOnString) throws Exception {
+        return getSurveyV3(surveyGuid, createdOnString);
+    }
+    @Deprecated
+    public Result getSurveyMostRecentVersionV1(final String surveyGuid) throws Exception {
+        return getSurveyMostRecentVersionV3(surveyGuid);
+    }
+    @Deprecated
+    public Result getSurveyMostRecentlyPublishedVersionV1(final String surveyGuid) throws Exception {
+        return getSurveyMostRecentlyPublishedVersionV3(surveyGuid);
+    }
+    @Deprecated
+    public Result getMostRecentPublishedSurveyVersionByIdentifierV1(String identifier) throws Exception {
+        return getMostRecentPublishedSurveyVersionByIdentifierV3(identifier);
+    }
+    @Deprecated
+    public Result deleteSurveyV1(String surveyGuid, String createdOnString) throws Exception {
+        return deleteSurveyV3(surveyGuid, createdOnString);
+    }
+    @Deprecated
+    public Result getSurveyAllVersionsV1(String surveyGuid) throws Exception {
+        return getSurveyAllVersionsV3(surveyGuid);
+    }
+    @Deprecated
+    public Result createSurveyV1() throws Exception {
+        return createSurveyV3();
+    }
+    @Deprecated
+    public Result versionSurveyV1(String surveyGuid, String createdOnString) throws Exception {
+        return versionSurveyV3(surveyGuid, createdOnString);
+    }
+    @Deprecated
+    public Result updateSurveyV1(String surveyGuid, String createdOnString) throws Exception {
+        return updateSurveyV3(surveyGuid, createdOnString);
+    }
+    @Deprecated
+    public Result publishSurveyV1(String surveyGuid, String createdOnString) throws Exception {
+        return publishSurveyV3(surveyGuid, createdOnString);
+    }
+    
+    private void canAccessSurvey(UserSession session) {
+        boolean isDeveloper = session.getUser().isInRole(DEVELOPER);
+        boolean isConsentedUser = session.getUser().doesConsent();
+        
+        if (isDeveloper || isConsentedUser) {
+            return;
+        }
+        // An imperfect test, but normal users have no other roles, so for them, access 
+        // is restricted because they have not consented.
+        Set<Roles> roles = new HashSet<>(session.getUser().getRoles());
+        roles.remove(TEST_USERS);
+        if (session.getUser().getRoles().isEmpty()) {
+            throw new ConsentRequiredException(session);
+        }
+        // Otherwise, for researchers and administrators, the issue is one of authorization.
+        throw new UnauthorizedException();        
```

Comment:
Thinking through this a little bit more, in all our other APIs (like upload), if a researcher or an admin tried to upload without consenting, they would get a ConsentRequiredException. So based on that precendent, if a non-Developer tried to get a survey and weren't consented, they should get ConsentRequiredException even if they were a developer or a researcher.

On the other hand, if we want to change the semantics so that researchers and administrators never consent (which as a side effect, they can't) and can never call "normal user" APIs, we might want to consider refactoring the TEST_USER (which appears to be an artifact of our automated tests, and not of any manual QA process outside of Sage), to make this test cleaner.

On the other other hand, this might be a side effect of researchers and admins considered to be always consented (will need to double check on that), which means they always had access to this API before, and now they don't.